error: no field named 'struct' in union 'builtin.Type'

Tried to build the project in `0.14.0-dev.121+ab4c461b7`

```
install
└─ install poop
   └─ zig build-exe poop Debug native 1 errors
src/main.zig:379:57: error: no field named 'struct' in union 'builtin.Type'
            inline for (@typeInfo(Command.Measurements).@"struct".fields) |field| {
                                                        ^~~~~~~~~
/opt/zig/bin/lib/std/builtin.zig:256:18: note: union declared here
pub const Type = union(enum) {
                 ^~~~~
referenced by:
    posixCallMainAndExit: /opt/zig/bin/lib/std/start.zig:515:32
    _start: /opt/zig/bin/lib/std/start.zig:342:40
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
error: the following command failed with 1 compilation errors:
/opt/zig/bin/zig build-exe -ODebug -Mroot=/home/panda/code/poop/poop/src/main.zig --cache-dir /home/panda/code/poop/poop/.zig-cache --global-cach
e-dir /home/panda/.cache/zig --name poop --listen=- 
Build Summary: 0/3 steps succeeded; 1 failed (disable with --summary none)
install transitive failure
└─ install poop transitive failure
   └─ zig build-exe poop Debug native 1 errors
error: the following build command failed with exit code 1:
/home/panda/code/poop/poop/.zig-cache/o/7dfe44708378c6fc7fcbba03bed2a051/build /opt/zig/bin/zig /home/panda/code/poop/poop /home/panda/code/poop/
poop/.zig-cache /home/panda/.cache/zig --seed 0x8f5ee784 -Z2bc430c26a3df3cf
```

--- a/src/main.zig
+++ b/src/main.zig
@@ -376,7 +376,7 @@ pub fn main() !void {
 
             try stdout_w.writeAll("\n");
 
-            inline for (@typeInfo(Command.Measurements).@"struct".fields) |field| {
+            inline for (@typeInfo(Command.Measurements).Struct.fields) |field| {
                 const measurement = @field(command.measurements, field.name);
                 const first_measurement = if (command_n == 1)
                     null

